server/router.go

```diff
@@ -26,6 +37,9 @@ func InitRouter() *echo.Echo {
 	e.Use(middleware.Recover())
 	e.Use(redoc.Serve(redoc.Opts{Title: "CyVerse Quota Management System"}))
 
+	// Register a custom validator.
+	e.Validator = &CustomValidator{validator: validator.New()}
+
```

Comment:
Adding a custom validator makes it easier to do additional validation checks on incoming request bodies.